docs/getting_started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started with ComposeDB
+
+## Create a schema
+
+* Write a graphQL compatible composite schema in a text file, such as
+
+example: task.schema
+```
+type Task @model(
+    accountRelation: LIST,
+    description: "Basic task"
+) {
+    assignee: String! @length(min:1, max: 30)
+    completed: Boolean
+    content: String! @length(max:2000)
+}
+
+```
+
+* generate a DID key 
+
+If you do not already have a DID key handy, you will need to create one 
+
+If you have downloaded and installed this repo or the [js-glaze](https://github.com/ceramicstudio/js-glaze) repo you can run
```

Comment:
there is inbuilt function to create did in composeDB itself 'composedb did:COMMAND'